### PR TITLE
i18n: Update translations from transifex

### PR DESCRIPTION
**Pull request by lukaw3d**
Supersedes and closes https://github.com/oasisprotocol/oasis-wallet-web/pull/810

### CODE REVIEW

**Review by tjanez — APPROVED** [review]

**Comment by ysz** [comment]
`StakingWithdraw` transaction type changes seem mixed with translation changes. Should it be separate PR?

File: src/locales/fr/translation.json [hunk 1]
```diff
@@ -1,115 +1,67 @@
...
-    "wallet": "Portefeuille",
-    "stake": "Staking",
-    "closeWallet": "Refermer le portefeuille"
-  },
-  "home": {
-    "existing": {
-      "header": "Ouvrir un portefeuille existant",
-      "description": "Ouvrez votre portefeuille en utilisant un Ledger, votre clé privée, ou votre phrase secrète.",
-      "button": "Ouvrir mon portefeuille"
+  "account": {
```
[the remark this answers is not in the document]
Comment (lukaw3d) [reply src/locales/fr/translation.json:2]: Transifex. It looks like it orders to match english file

**Comment by lukaw3d** [comment]
@ysz Sure; added https://github.com/oasisprotocol/oasis-wallet-web/pull/819

**Review by Hasancan2 — COMMENTED** [review]
Us

**Pull request merged**